backend/oasst_backend/api/v1/tasks.py

```diff
@@ -204,7 +204,7 @@ def tasks_acknowledge_failure(
         raise OasstError("Failed to not acknowledge task.", OasstErrorCode.TASK_NACK_FAILED)
 
 
-@router.post("/interaction")
+@router.post("/interaction", response_model=protocol_schema.TaskDone)
```

Comment:
@yk note on how to specify multiple possible response_models: https://stackoverflow.com/questions/72919006/is-there-any-way-to-have-multiple-response-models-in-fastapi-openapi